src/Compilers/CSharp/Test/Semantic/Semantics/RecordTests.cs

```diff
@@ -4662,6 +4713,166 @@ public void CopyCtor_InaccessibleInMetadata()
                 );
         }
 
+        [Fact, WorkItem(45077, "https://github.com/dotnet/roslyn/issues/45077")]
+        public void CopyCtor_AmbiguitiesInMetadata()
```

Comment:
>CopyCtor_AmbiguitiesInMetadata [](start = 20, length = 30)

It is impossible to follow what this test is doing. I believe clarity of tests is more important than brevity and extreme code reuse.